package.json

```diff
@@ -59,6 +60,6 @@
     "@aries-framework/react-native@0.4.0": "patch:@aries-framework/react-native@npm:0.3.3#./.yarn/patches/@aries-framework-react-native-npm-0.3.3-bb75ece22d.patch",
     "@aries-framework/anoncreds@^0.4.0": "patch:@aries-framework/anoncreds@npm%3A0.4.0#./.yarn/patches/@aries-framework-anoncreds-npm-0.4.0-4d3b4e769d.patch",
     "@aries-framework/anoncreds@0.4.0": "patch:@aries-framework/anoncreds@npm%3A0.4.0#./.yarn/patches/@aries-framework-anoncreds-npm-0.4.0-4d3b4e769d.patch",
-    "@aries-framework/indy-vdr@^0.4.0": "patch:@aries-framework/indy-vdr@npm%3A0.4.0#./.yarn/patches/@aries-framework-indy-vdr-npm-0.4.0-b01067562c.patch"
+    "@aries-framework/indy-vdr@^0.4.0": "patch:@aries-framework/indy-vdr@patch%3A@aries-framework/indy-vdr@npm%253A0.4.0%23./.yarn/patches/@aries-framework-indy-vdr-npm-0.4.0-b01067562c.patch%3A%3Aversion=0.4.0&hash=0e7997&locator=bc-wallet-mobile%2540workspace%253A.#./.yarn/patches/@aries-framework-indy-vdr-patch-28d47565c5.patch"
```

Comment:
This patch looks different than all the rest, is there something special about it? Maybe we need to delete the existing patch and then add the new one back afterwards?

Reply:
resolved